### PR TITLE
About ch

### PR DESCRIPTION
**Pull request by Moamin-AbuEwaida**
adding the HYF team part to the about page

### CODE REVIEW

**Review by bringmetheaxx — APPROVED** [review]

File: css/styles.css [hunk 1]
```diff
@@ -1580,3 +1580,68 @@ ul {
...
+  font-size: 24px;
+  color: rgb(28, 34, 148);
+  margin: 0 40px;
+}
+
+.ourteam_coaches {
+  display: flex;
+  overflow: auto;
+  position: relative;
+}
```
Comment (danielhalasz) [inline css/styles.css:1604]: justify-content: space-between;

File: css/styles.css [hunk 2]
```diff
@@ -1580,3 +1580,68 @@ ul {
...
+  margin-bottom: 10px;
+}
+
+.ourteampic {
+  height: 180px;
+  width: 180px;
+  filter: grayscale(100%);
+  border-radius: 100%;
+  margin: 20px auto;
+}
```
Comment (danielhalasz) [inline css/styles.css:1638]: object-fit: cover;

**Review by danielhalasz — CHANGES_REQUESTED** [review]

**Comment by Moamin-AbuEwaida** [comment]
just added both comments @danielhalasz

**Review by danielhalasz — APPROVED** [review]
THX

**Pull request merged**